src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -29,24 +29,28 @@ const FormSection = class FormSection {
 }
 
 const FormField = class FormField {
-    constructor(label, required, value, allowDefaultValue, sectionId, defaultValue) {
+    constructor(label, required, value, allowDefaultValue, sectionId, defaultValue, dataType, picklistOptions) {
```

Comment:
I'd like to continue our conversation internally about using the class keyword and constructors vs using objects directly - not sure how much gain we achieve by having this class definition code included.

Reply:
All set!

Reply:
Awesome - looks good!